### ISSUE TITLE
The reason why some results are gray

### ISSUE THREAD

**Issue by justinday123** [foivospar/Arc2Face#42]
![Image](https://github.com/user-attachments/assets/e5dc2068-db65-4488-ae12-08e078818cb1)

Some results such as figure 18 (and other results), arc2face generates images in grayscale. What is the reason? I think the reason is normalizing arcface's embedding, using this code.

id_emb = id_emb/torch.norm(id_emb, dim=1, keepdim=True)   # normalize embedding

Is normalize is essential? Can't I make the results without grayscale?

**Comment by samfxbhgh** [comment]
Maybe I think it is caused by the training dataset.  You can see AgeDB400, there are many images with grayscale.

**Comment by foivospar** [comment]
Yes, it is because of the training images in WebFace (AgeDB was only used for evaluation).
